addons/website_portal/controllers/main.py

```diff
@@ -77,6 +77,14 @@ def details_form_validate(self, data):
             if not data.get(field_name):
                 error[field_name] = 'missing'
 
+        in_europe = False
+        if data.get('country_id'):
+            europe = request.env["res.country.group"].search([('name', '=', 'Europe')])
```

Comment:
You may want to use ```request.env.ref('base.europe')``` instead as the name can be renamed. Also it is possible that the country group has been deleted, so combining it with the search as a fallback might improve the quality of this method.

Reply:
@wtaferner Thanks